Application/EdFi.Ods.AdminApi/Features/OdsInstanceContext/AddOdsInstanceContext.cs

```diff
@@ -78,7 +78,7 @@ private bool BeAnExistingOdsInstance(int id)
         private bool BeUniqueCombinedKey(AddOdsInstanceContextRequest request)
         {
             return !_getOdsInstanceContextsQuery.Execute().Any
-                (x => x.OdsInstanceId == request.OdsInstanceId &&
+                (x => x.OdsInstanceId() == request.OdsInstanceId &&
                 x.ContextKey.Equals(request.ContextKey, StringComparison.OrdinalIgnoreCase));
```

Comment:
@jagudelo-gap 
Can we use x.OdsInstance?.OdsInstanceId instead of extension method.

Reply:
Done